test/nwise.js

```diff
@@ -0,0 +1,22 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('nwise', function() {
+    var sevenLs = [1, 2, 3, 4, 5, 6, 7];
+    it('creates a list of n-tuples from a list', function() {
+        assert.deepEqual(R.nwise(3, sevenLs), [[1, 2, 3], [2, 3, 4], [3, 4, 5], [4, 5, 6], [5, 6, 7]]);
+        assert.deepEqual(R.nwise(4, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    });
+
+    it('returns an empty list when `n` > `list.length`', function() {
+        assert.deepEqual(R.nwise(6, [1, 2, 3]), []);
```

Comment:
for `chunk` I agree. for `nwise` that could only be an issue when the list length is less than the tuple size. in that case reconstructing the missing tuple is trivial, since it is the input list